MAINTENANCE.md

```diff
@@ -19,3 +29,20 @@ To generate all the artifacts above, one proceeds as follows:
 6. `git push --tags` to push the new tag.
 7. Go to the newly created release page on GitHub and edit it by pressing the "Generate Release Notes" and the `@` button. Save the release.
 
+#### Alternative Process: `cargo smart-release`
+
+This also maintains the `changelog.md` file, and it's able to pick up previous `cargo publish` releases and thus mixes well.
+However, it's quite new and needs some polishing.
+
+To just update the changelog, run `cargo changelog --write`.
+
+To create a new release, do as follows:
+
+1. `git checkout -b release-<next-version>` - move to a branch to prepare making changes to the repository. *Changes cannot be made to `main` as it is protected.*
+2. `cargo changelog --write` to update the changelog. Edit it to your liking.
+3. Create a commit like `git commit -am "update changelog prior to release"`
+4. `cargo smart-release -e --update-crates-index` - double-check that the release version is sensible, and if not, just edit `Cargo.toml` by hand before.
+5. Tags were created, and they should be pushed, but it might be that it fails to create a GitHub release. 
+6. Create an additional release tag with the `v` removed, i.e. `git tag -s <version> v<version>`. Then `git push  --tags` by hand.
+7. Double-check the newly created release on GitHub is populated - if not, generate the changelog manually.
```

Comment:
It looks like the release for 1.0.31 was not populated. Can you describe the steps more clearly here, perhaps by doing them for 1.0.31?